pylint/checkers/variables.py

```diff
@@ -2666,17 +2737,19 @@ def _check_unpacking(
         values = self._nodes_to_unpack(inferred)
         if values is not None:
             if len(targets) != len(values):
-                # Check if we have starred nodes.
-                if any(isinstance(target, nodes.Starred) for target in targets):
-                    return
+                if isinstance(inferred, DICT_TYPES):
+                    msg_id = "unbalanced-dict-unpacking"
```

Comment:
```suggestion
                    symbol = "unbalanced-dict-unpacking"
```
Technically, msgid is the C1234 and symbol is "msg-name". At least that's the standard in ``pylint.messages``